ansible_community_steering_committee.rst

```diff
@@ -45,50 +42,36 @@ Members:
-If you want to discuss an idea that can impact the community, collection best practices, inclusion policy, submitted new Policy/Proposals & New Collection Inclusion Requests or other topics the steering committee is responsible for, create a new issue in the `community-topics repository <https://github.com/ansible-community/community-topics>`_ as a discussion topic, and it’ll be discussed asynchronously in the issue itself.
+* New collection inclusion requests are submitted through a new discussion in the `ansible-inclusion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_ repository.
 
-Once the filed issue topic is thorough, the Steering Committee will
-discuss the final conclusions and then will vote directly on the issue
+Depending on a topic you want to discuss with the Community and the Committee, as you prepare your proposal, please consider the requirements established by:
 
-Depending on the policy/collection inclusion requests/ideas you propose, as you prepare your proposal you may need to consider requirements pertaining to research involving `Ansible Community Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#code-of-conduct>`_, `Ansible Collection Inclusion Checklist <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_ and `Ansible Community Guide <https://docs.ansible.com/ansible/latest/community/index.html>`_.
+* `Ansible Community Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html#code-of-conduct>`_.
+* `Ansible Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_.
+* `Ansible Collection Inclusion Checklist <https://github.com/ansible-collections/overview/blob/main/collection_checklist.md>`_.
 
 Meetings:
 =========
 
 See the Community Working Group meeting `schedule <https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays>`_.
 
-Agenda
-~~~~~~
-
-You can find the discussion topics in the `community-topics <https://github.com/ansible-community/community-topics>`_ repository.
-
-For topics that are going to be discussed in the next meeting, see `here <https://github.com/ansible-community/community-topics/labels/next_meeting>`_.
-
-For a short summary on the past topics already discussed in the Community Meeting, see the `Meeting Agenda issue <https://github.com/ansible/community/issues/539>`_.
-
-Minutes of Meeting
-~~~~~~~~~~~~~~~~~~
+Meeting summaries are posted in the `Community Working Group Meeting Agenda <https://github.com/ansible/community/issues/539>`_ issue.
```

Comment:
Issue #539 tracked 2021 agendas.
Issue [#645](https://github.com/ansible/community/issues/645) tracks the agenda for meetings in 2022.

Reply:
Good catch! I've updated it with a link that uses labels, so it will always point to the current issue (i.e. there will be no need to update it every year). Thanks!